### PR TITLE
donwgrading material ui to fix issue 45

### PR DESCRIPTION
**Pull request by padraigobrien**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "@backstage/core": "^0.6.0",
     "@backstage/plugin-catalog": "^0.2.9",
     "@backstage/theme": "^0.2.1",
-    "@material-ui/core": "^4.11.3",
+    "@material-ui/core": "^4.9.1",
```
Comment (iain-b) [inline package.json:33]: 4.11.0 is used in core and the other plugins. Might be best to use that for the sake of consistency

Comment (padraigobrien) [reply package.json:33]: Will do

**Comment by anderoo** [comment]
Many thanks for this 👏

File: package.json (hunk 1 above)
Comment (padraigobrien) [reply package.json]: done

**Pull request merged**